.github/workflows/semantic-pr.yml

```diff
@@ -0,0 +1,103 @@
+---
+name: "Semantic PR"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
```

Comment:
What does `synchronize` indicate / can you point me to a doc with the different types? I had trouble finding one last time I looked

Reply:
Here is the extensive list. https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request
Here is a shortlist with meaning https://frontside.com/blog/2020-05-26-github-actions-pull_request/
Synchronize: 	commit(s) pushed to the pull request

commit(s) pushed to the pull request
--





commit(s) pushed to the pull request
--





commit(s) pushed to the pull request
--